packages/core/src/queries/user.ts

```diff
@@ -60,11 +60,17 @@ export const userSearchFields = Object.freeze(Object.values(pick(Users.fields, .
 
 export const createUserQueries = (pool: CommonQueryMethods) => {
   const findUserByUsername = async (username: string) =>
-    pool.maybeOne<User>(sql`
-      select ${sql.join(Object.values(fields), sql`,`)}
-      from ${table}
-      where ${fields.username}=${username}
-    `);
+    process.env.CASE_INSENSITIVE_USERNAME === '1'
```

Comment:
We manage the environment values in "core/src/env-set" instead of using `process.env` directly, would you mind taking a look and see if it is possible to add this value to the place?

Reply:
Oh I see!
Added as `EnvSet.values.isCaseInsensitiveUsername`